x.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Please revert this change. There are systems that do not have python3 installed (https://github.com/rust-lang/rust/issues/71818#issuecomment-623291126). If you have trouble locally you can make a symbolic link from `python3` to `python`.
```suggestion
 #!/usr/bin/env python
```